media-proxy/src/proxy_context.cc

```diff
@@ -721,10 +719,9 @@ int ProxyContext::RxStart(const RxControlRequest* request)
 
     if (mDevHandle == NULL) {
         struct mtl_init_params st_param = {};
-        ParseStInitParam(request, &st_param);
-
         /* set default parameters */
-        st_param.flags = MTL_FLAG_BIND_NUMA;
+        st_param.flags |= MTL_FLAG_BIND_NUMA;
```

Comment:
```suggestion
```
You don't need this line since `st_param.flags` is being overwritten later in `ParseStInitParam()`